misc/acrn-config/board_config/misc_cfg_h.py

```diff
@@ -117,6 +121,9 @@ def generate_file(config):
     print("{0}".format(board_cfg_lib.HEADER_LICENSE), file=config)
     print("{}".format(MISC_CFG_HEADER), file=config)
 
+    # define CONFIG_MAX_PCPCU_NUM
+    print("#define CONFIG_MAX_PCPU_NUM\t{}U".format(max_cpu_num), file=config)
```

Comment:
CONFIG_MAX_PCPU_NUM is still used, I don't understand the commit message "to instead of CONFIG_MAX_PCPU_NUM"

Reply:
You are right, this commit message is not update to latest.
i will update it soon.